doc/installation.rst

```diff
@@ -131,6 +131,11 @@ macOS.
 It might be possible to use a version of gcc built using Homebrew or
 MacPorts, but this is untested and unsupported.
 
+It appears that on some versions, /usr/local is not in the default
+compiler paths so you might need to add ``-L /usr/local/lib -I
+/usr/local/include`` to the setup.py command.
```

Comment:
I think it is only for the build command. So I would write:

to the ``python setup.py build`` command.

When you amend a commit, this don't trigger a notification. So I just saw this change now.